src/index.html

```diff
@@ -7,8 +7,30 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
   <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header class="header">
+      <div class="logo">
+        <a href="#">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <a href="#" class="nav__link is-active">Apple</a>
+        <a href="#" class="nav__link">Samsung</a>
+        <a href="#" class="nav__link">Смарфоны и телефоны</a>
```

Comment:
```suggestion
        <a href="#" class="nav__link">Смартфоны и телефоны</a>
```

https://user-images.githubusercontent.com/38065883/124469060-8b832400-dda2-11eb-8b14-fb26b1f7068f.mov

Reply:
К сожалению, видео не помогло мне разобраться с расхождениями в отступах. 
Переписала код по новой, но все равно не получается пройти тестирование без ошибок.
По тесту вижу, что есть расстояние между  nav и ul, но не пойму, что мне сделать, чтобы от него избавится. Пробовала обнулить у первого ребенка margin –  не помогло.

https://obordiug.github.io/layout_moyo-header/
https://obordiug.github.io/layout_moyo-header/report/html_report/

Reply:
можна ще обнулити марджини в ul і li